kview/src/main/java/dev/ikm/komet/kview/klfields/readonly/ReadOnlyKLFieldFactory.java

```diff
@@ -60,93 +48,93 @@ public Node createReadOnlyDiTree(ViewProperties viewProperties, FieldRecord<?> f
-            // update field's purpose
-            Label fieldValue = (Label) componentRowItem.lookup(".semantic-field-value");
-            EntityFacade value = component.get().entity();
-            fieldValue.setText(text(viewProperties, value.nid()));
-            return componentRowItem;
-        }
-        return null;
-    }
+//    /**
+//     * Creates a read-only component. Such as language value is English Language.
+//     * @param fieldRecord
+//     * @return JavaFX Node representing a component for semantic details.
+//     */
+//    public Node createReadOnlyComponent(ViewProperties viewProperties, FieldRecord<?> fieldRecord) {
+//        JFXNode<Node, Void> jfxNode = FXMLMvvmLoader.make(getResource("/dev/ikm/komet/kview/controls/read-only-component-field.fxml"));
+//        Node componentRow = jfxNode.node();
+//        // update field's meaning title label
+//        Label fieldMeaning = (Label) componentRow.lookup(".semantic-field-type-label");
+//        fieldMeaning.setTooltip(new Tooltip(text(viewProperties, fieldRecord.purposeNid())));
+//        fieldMeaning.setText(text(viewProperties, fieldRecord.meaningNid()));
+//
+//        // update identicon
+//        ImageView identicon = (ImageView) componentRow.lookup(".identicon-image-view");
+//        identicon.setImage(Identicon.generateIdenticonImage(fieldRecord.purpose().publicId()));
+//
+//        // update field's purpose
+//        Label fieldValue = (Label) componentRow.lookup(".semantic-field-value");
+//        EntityFacade value = (EntityFacade) fieldRecord.value();
+//        fieldValue.setText(value.description());
+//        return componentRow;
+//    }
+//    public Node createReadOnlyComponentSet(ViewProperties viewProperties, FieldRecord<?> fieldRecord) {
+//        JFXNode<Node, Void> jfxNode = FXMLMvvmLoader.make(getResource("/dev/ikm/komet/kview/controls/read-only-component-set-field.fxml"));
+//        Node componentRow = jfxNode.node();
+//        // obtain vbox to add items from set
+//        VBox container = (VBox) componentRow.lookup(".semantic-field-collection-container");
+//
+//        // update field's meaning title label
+//        Label fieldMeaning = (Label) componentRow.lookup(".semantic-field-type-label");
+//        fieldMeaning.setTooltip(new Tooltip(text(viewProperties, fieldRecord.purposeNid())));
+//        fieldMeaning.setText(text(viewProperties, fieldRecord.meaningNid()));
+//
+//        // loop through all components
+//        IntIdSet componentSet = (IntIdSet) fieldRecord.value();
+//        componentSet.forEach(componentId -> {
+//            Latest<EntityVersion> component = viewProperties.calculator().stampCalculator().latest(componentId);
+//            component.ifPresent(entityVersion -> {
+//                Node componentRow2 = createReadOnlyComponentListItem(viewProperties, component);
+//                container.getChildren().add(componentRow2);
+//            });
+//        });
+//        return componentRow;
+//    }
```

Comment:
I think we should remove these commented out functions. :-)
Good call btw, regrading DiTree and DiGraph. Once we create these controls we can remove class.

Reply:
Agree, it's better and cleaner if we just remove the commented-out code. 
Just did that and pushed.